velox/docs/functions/spark/array.rst

```diff
@@ -100,6 +100,15 @@ Array Functions
         SELECT filter(array(0, 2, 3), (x, i) -> x > i); -- [2, 3]
         SELECT filter(array(0, null, 2, 3, null), x -> x IS NOT NULL); -- [0, 2, 3]
 
+.. function:: flatten(array(array(E))) -> array(E)
+
+    Transforms an array of arrays into a single array.
+    Returns NULL if the input is NULL or the input array contains any NULL element. ::
```

Comment:
> or the input array contains any NULL element.

Give below example, this description seems to be inaccurate as NULL is appended just like a valid element.

> SELECT flatten(array(array(1, 2), array(3, NULL))); -- [1, 2, 3, NULL]

Reply:
> Give below example, this description seems to be inaccurate as NULL is appended just like a valid element.

The description might be confusing, let me change it as `Returns NULL if the input is NULL or any of the input arrays is NULL.`
Means that we only check the element of the top level array.